skbio/sequence/_sequence.py

```diff
@@ -520,8 +524,8 @@ def _string(self):
         return self._bytes.tostring()
 
     @stable(as_of="0.4.0")
-    def __init__(self, sequence, metadata=None,
-                 positional_metadata=None):
+    def __init__(self, sequence, metadata=None, positional_metadata=None,
+                 lowercase=False):
```

Comment:
the order of constructor parameters changed, please update DNA, RNA, and Protein class docstrings' Parameters section